common-js/Biconomy.js

```diff
@@ -1854,6 +1854,11 @@ function _getParamValue(paramObj) {
   var value;
 
   try {
+    if (paramObj && paramObj.type == "bytes" && (!paramObj.value || paramObj.value == undefined || paramObj.value == null)) {
```

Comment:
in this case would the value of **paramObj.value** be an empty array?

Reply:
It will be undefined
fault is in abi.decoder library. when one of your function parameters in the smart contract is bytes and say if its optional you're gonna send 0x / defaults to 0x. The library can't decode it as empty bytes string and outputs undefined.